src/main/java/com/desiato/puresynth/validators/AuthenticationRequestValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.desiato.puresynth.validators;
+
+import com.desiato.puresynth.dtos.AuthenticationRequestDTO;
+import com.desiato.puresynth.exceptions.ValidationException;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class AuthenticationRequestValidator {
```

Comment:
Both validator classes could inherit from e.g. abstract class. The abstract class could handle management of errorMessages: e.g. adding and throwing exception.